advanced/editor-command-identifiers.md

```diff
@@ -7,93 +7,397 @@ description: The complete list of exposed editor commands.
+tinymce.activeEditor.execCommand('Delete')
+tinymce.activeEditor.execCommand('ForwardDelete')
+tinymce.activeEditor.execCommand('mceNewDocument')
+tinymce.activeEditor.execCommand('Redo')
+tinymce.activeEditor.execCommand('Undo')
+tinymce.activeEditor.execCommand('mceAddUndoLevel')
+tinymce.activeEditor.execCommand('mceEndUndoLevel')
+tinymce.activeEditor.execCommand('mceCleanup')
+tinymce.activeEditor.execCommand('mceSelectNode', false, '<DOM_node>')
+tinymce.activeEditor.execCommand('mceSelectNodeDepth', false, 2) // For two nodes up.
+tinymce.activeEditor.execCommand('mceRemoveNode') /* OR */
+tinymce.activeEditor.execCommand('mceRemoveNode', false, '<DOM_node>')
+```
+
+## Plugin Commands
+
+Commands are available for the following plugins:
+
+- [Advanced Lists](#advancedlists)
+- [Advanced Tables](#advancedtables)
+- [Anchor](#anchor)
+- [Autoresize](#autoresize)
+- [Case Change](#casechange)
+- [Character Map](#charactermap)
+- [Code](#code)
+- [Code Sample](#codesample)
+- [Comments](#comments)
+- [Directionality](#directionality)
+- [Format Painter](#formatpainter)
+- [Full Page](#fullpage)
+- [Full Screen](#fullscreen)
+- [Help](#help)
+- [Horizontal Rule](#horizontalrule)
+- [Image](#image)
+- [Image Tools](#imagetools)
+- [Insert Date/Time](#insertdatetime)
+- [Link](#link)
+- [Lists](#lists)
+- [Media](#media)
+- [Nonbreaking Space](#nonbreakingspace)
+- [Page Break](#pagebreak)
+- [Paste](#pasteorpowerpaste)
+- [Permanent Pen](#permanentpen)
+- [PowerPaste](#powerpaste)
+- [Preview](#preview)
+- [Print](#print)
+- [Save](#save)
+- [Search and Replace](#searchandreplace)
+- [Spell Checker](#spellchecker)
+- [Table](#table)
+- [Template](#template)
+- [Table of Contents](#tableofcontents)
+- [Visual Blocks](#visualblocks)
+- [Visual Characters](#visualcharacters)
+
+### Advanced Lists
+
+The following commands require the [Advanced Lists (`advlist`)]({{ site.baseurl }}/plugins/advlist/) plugin.
+
+{% include commands/advlist-cmds.md %}
+
+### Advanced Tables
+
+The following commands require the [Advanced Tables (`advtable`)]({{ site.baseurl }}/plugins/advtable/) plugin.
+
+{% include commands/advtable-cmds.md %}
+
+### Anchor
+
+The following command requires the [Anchor (`anchor`)]({{ site.baseurl }}/plugins/anchor/) plugin.
+
+{% include commands/anchor-cmds.md %}
+
+### Autoresize
+
+The following command requires the [Autoresize (`autoresize`)]({{ site.baseurl }}/plugins/autoresize/) plugin.
+
+{% include commands/autoresize-cmds.md %}
+
+### Case Change
+
+The following commands require the [Case Change (`casechange`)]({{ site.baseurl }}/plugins/casechange/) plugin.
+
+{% include commands/casechange-cmds.md %}
+
+### Character Map
+
+The following command requires the [Character Map (`charmap`)]({{ site.baseurl }}/plugins/charmap/) plugin.
+
+{% include commands/charmap-cmds.md %}
+
+### Code
+
+The following command requires the [Code (`code`)]({{ site.baseurl }}/plugins/code/) plugin.
```

Comment:
These commands apply to both `code` and `advcode`.

Reply:
Fixed